src/SFML/Window/WindowImpl.cpp

```diff
@@ -109,6 +109,16 @@ void WindowImpl::setJoystickThreshold(float threshold)
 
 
 ////////////////////////////////////////////////////////////
+bool WindowImpl::setFileDroppingEnabled(bool enabled)
+{
+    // This is the fallback for unsupported platforms.
+    // Trying to enable file dropping returns a bad value
+    // indicating an error, whereas disabling it is fine.
+    return !enabled;
```

Comment:
Ah, okay, from that perspective it makes sense. But I'm not 100% sure how it should react considering it's not implemented/available. You're making it fail silently.